### PR TITLE
feat(objectValuesToString): add 'objectValuesToString' method with tests

### LINKED ISSUE #33: Add a objectValuesToString function
Hi!

I would like to add a new function.
It would a take in parameter an object and return a String that is composed of all its properties values separated by comas.

I found this solution on [stack overflow](http://stackoverflow.com/questions/31017475)

Would it be ok ?

### PR DESCRIPTION
**Pull request by DdZ-Fred**
To close #33: I've added a function that takes in parameter an object and returns a comma-separated

string that is composed of all its values. The tests make sure the value passed is really an object

and that its undefined or null properties are skipped

#33

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,4 +13,5 @@ export {
   padLeft,
   randomInteger,
   arrayFill,
+  objectValuesToString
```
Comment (kentcdodds) [inline src/index.js]: Please leave a dangling comma here to keep our git history clean :) read [more](http://eslint.org/docs/rules/comma-dangle).

**Comment by kentcdodds** [comment]
This looks great! Thanks! It looks like there are merge issues with this. Could you [rebase your branch](https://egghead.io/lessons/javascript-how-to-rebase-a-git-pull-request-branch?course=how-to-contribute-to-an-open-source-project-on-github) onto master?

File: src/index.js (hunk 1 above)
Comment (DdZ-Fred) [inline src/index.js]: I just saw you created you own config for eslint and that is was actually specified in the package.json! my mistake, I always work with an .eslintrc.json at the root of a project and, as I couldn't see any, I assumed I had to create one and use the default eslint:recommended.

**Comment by DdZ-Fred** [comment]
Ok, I did the rebase but now a problem occurs when a try to run the test:watch.

'ava --watch' doesn't recognise the es6 import...did I miss something ?

```
λ npm run test:watch

> stack-overflow-copy-paste@0.0.0-semantically-released test:watch D:\_DEV_\stack-overflow-copy-paste
> ava --watch

D:\_DEV_\stack-overflow-copy-paste\src\index.js:1
(function (exports, require, module, __filename, __dirname) { import flatten from './flatten'
                                                              ^^^^^^
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:513:28)
```

I took the opportunity to replace all the ava same() calls by deepEqual() also.

**Comment by kentcdodds** [comment]
Thanks for reporting that! Could you try rebasing again? I've fixed the problem.

**Comment by DdZ-Fred** [comment]
I did the rebase and re-applied my changes. Though, I did a new pull-request which is available here #35

**Pull request closed without merging**